wait/host_port.go

```diff
@@ -88,13 +88,15 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 	//internal check
 	command := buildInternalCheckCommand(hp.Port.Int())
 	for {
-		exitCode, err := target.Exec(ctx, []string{"/bin/bash", "-c", command})
+		exitCode, err := target.Exec(ctx, []string{"/bin/sh", "-c", command})
 		if err != nil {
 			return errors.Wrapf(err, "host port waiting failed")
 		}
 
 		if exitCode == 0 {
 			break
+		} else if exitCode == 126 {
+			return errors.New("/bin/sh command not executable")
```

Comment:
what would the `err` variable have in this case? I am not sure without testing...but would it be better to
```go
if exitCode == 0 {
    break
} else {
    return fmt.Errorf("received exit code %d when running /bin/sh, error: %s", exitCode, err)
}
```
or something similar?  that way we handle for any exit code that is not a 0

Reply:
`err` would be nil, otherwise L92  will capture it

Reply:
ah I missed that, thanks!